services/viva/microservice/src/lambdas/syncWorkflow.ts

```diff
@@ -104,22 +101,28 @@ function updateCaseDetailsWorkflow(keys: CaseKeys, newWorkflow: VivaWorkflow) {
   return dynamoDb.call('update', updateParams);
 }
 
-export async function syncWorkflow(input: LambdaRequest, dependencies: Dependencies) {
+function isSetWorkflowId(caseItem: CaseItem) {
+  return !!caseItem.details.workflowId;
+}
+
+export async function syncWorkflow(
+  input: LambdaRequest,
+  dependencies: Dependencies
+): Promise<boolean> {
   const { personalNumber } = input.detail.user;
 
   const caseList = await dependencies.getCasesByStatusType(personalNumber, [
-    ACTIVE_ONGOING,
-    ACTIVE_SUBMITTED,
-    ACTIVE_PROCESSING,
+    'active',
     CLOSED_REJECTED_VIVA,
     CLOSED_PARTIALLY_APPROVED_VIVA,
   ]);
+
   const isEmptyCaseList = caseList.length === 0;
   if (isEmptyCaseList) {
     return true;
   }
 
-  const caseListWithWorkflowId = caseList.filter(caseItem => !!caseItem.details.workflowId);
+  const caseListWithWorkflowId = caseList.filter(isSetWorkflowId);
```

Comment:
`isSetWorkflowId` doesn't sound right...I get the idea but it reads poorly. Maybe something like `isWorkflowIdValid` or `hasWorkflowId`?